### DISCUSSION TITLE
Can someone tell me how to install it?

### DISCUSSION THREAD

**Discussion by CypiGames** [FreeRDP/FreeRDP#8970] [General]
I want to install it but i don't understand how, english is not my native language and i'm dumb. Can someone tell me in simple language how to install it please? 

ps. I'm using windows

**Comment by akallabeth** [comment]
you can download our nightly builds from https://github.com/FreeRDP/FreeRDP/wiki/PreBuilds or compile yourself.

no installation required, it is a simple `exe` file that can be run. 
that said, it is a program that is intended for specialized purposes (thinclients or similar), so you might be better off using the default RDP client if you want something with a UI (`wfreerdp.exe` is a command line application)

**Reply by CypiGames** [reply]
So i need to go to Windows section and download files from there and after that just open this exe?

**Reply by CypiGames** [reply]
I downloaded it and started wfreerdp.exe and it's not working. It doesn't pop up or show anything.

**Reply by akallabeth** [reply]
it is a command line application.
due to some windows design limitation it is not possible to have:
1. an application pop up a window without a `cmd` 
2. command line output on a `cmd` if started from one.

you can do something like `wfreerdp.exe <yourargs> >c:\tmp\somepath\logfile.txt 2>&1` and check the output in that file
